package.json

```diff
@@ -85,9 +85,9 @@
     "@formatjs/intl-pluralrules": "^5.0.2",
     "@metamask/detect-provider": "^1.2.0",
     "@reduxjs/toolkit": "^1.8.2",
-    "@shapeshiftoss/asset-service": "^7.2.0",
-    "@shapeshiftoss/caip": "^6.12.0",
-    "@shapeshiftoss/chain-adapters": "^7.15.1",
+    "@shapeshiftoss/asset-service": "^7.4.0",
+    "@shapeshiftoss/caip": "^6.13.0",
+    "@shapeshiftoss/chain-adapters": "^7.14.2",
```

Comment:
this is a downgrade

Reply:
OK now, checked against the published packages in https://github.com/shapeshift/lib/pull/955